template/module/README.rst

```diff
@@ -1,94 +1,20 @@
-.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
-   :target: https://www.gnu.org/licenses/agpl
-   :alt: License: AGPL-3
+**This file is going to be generated by oca-gen-addon-readme.**
 
-==============
-{module_title}
-==============
+*Manual changes will be overwritten.*
 
-This module extends the functionality of ... to support ...
-and to allow you to ...
+Please provide content in:
 
-Installation
-============
+* **DESCRIPTION.rst** (required)
+* INSTALL.rst (optional)
+* CONFIGURE.rst (optional)
+* **USAGE.rst** (optional, highly recommended)
+* DEVELOP.rst (optional)
+* ROADMAP.rst (optional)
+* **CONTRIBUTORS.rst** (optional, highly recommended)
+* CREDITS.rst (optional)
```

Comment:
HISTORY.rst (optional)?

Reply:
So far I did not include the changelog in the readme because I feared it would be too long. But we could. What do you prefer?

Reply:
The audience for a change log is users (bug fixes, improvements). A git commit history is too noisy for users (even developers using a library). There is always a lot of commits that do not belong to a changelog (eg refactorings), but are interesting for developers nevertheless. Moreover having several commits for a single features is often a desirable thing that also creates too much "noise" for a changelog.

So HISTORY.rst and the commit history are definitely two different things.

I would not add the link to the commit history in the README (the link to the repo is already a huge improvement and devs can easily find commits themselves).

I would love if we can gently push maintainers to start creating user oriented changelogs. That's why I also suggested to make it mandatory for mature addons.

Reply:
What about using github issues and milestones to manage the history? This would allow us to reference in the commit messages the context in which these changes occurs. From an issue we would also have the list of changes made to solve the problem. Thinking about it there are cases where issue and PR could be redundant and this would force us to create issues for different PR. This may not be the best idea, especially since this system will not work with our modules where we have 1 master branch per Odoo version. 
So, IMO, the best way to properly and efficiently manage the history is to have a HISTORY.rst file.

Reply:
OK, but if they are user oriented then you could include them in the README.rst, otherwise nobody will look at them.

Reply:
@StefanRijnhart @lmignon thanks for your input. I updated the generator to include HISTORY.rst. See the updated [example](https://github.com/acsone/mis-builder/blob/10.0-generated-readme/mis_builder/README.rst).

Reply:
@sbidoul Wouldn't it be more relevant to have the history at the end of the readme. It can become very long and reject the last two sections outside the reader's view.

Reply:
@lmignon could be. My reasoning was that HISTORY (past) and ROADME (future) fit well next to each other. Also having the even more important maintainers section at then makes it easily accessible by scrolling to the end.

Reply:
@sbidoul :+1: that makes sense...